modules/calibration/data/ch/velodyne_params/64E_S3_calibration_example.yaml

```diff
@@ -0,0 +1,243 @@
+lasers:
```

Comment:
At present, you should not need any configuration files of velodyne 64, because it has been duplicated with the default config.